webapp/portlet/src/main/webapp/vue-apps/common/components/ExoIdentitySuggester.vue

```diff
@@ -239,12 +245,13 @@ export default {
           if (!this.previousSearchTerm || this.previousSearchTerm !== this.searchTerm) {
             this.loadingSuggestions = 0;
             this.items = [];
-            this.$suggesterService.searchSpacesOrUsers(value,
+            this.$suggesterService.searchSpacesOrUsersOrGroups(value,
               this.items,
               this.typeOfRelations,
               this.searchOptions,
               this.includeUsers,
               this.includeSpaces,
+              this.includeGroups,
```

Comment:
It's better to not break the API by adding an additional parameter in the middle. Besides, you may need to add a new method inside `suggesterService.js` with a config object (instead of chained parameters with all options listed there) : 

this.$suggesterService.search({
  term: String,
  typeOfRelations: String,
  searchOptions: Object,
  includeUsers: Boolean,
  includeSpaces: Boolean,
  includeGroups: Boolean,
  onlyRedactor: Boolean,
  onlyManager: Boolean,
  loadingCallback: Function,
  successCallback: Function,
  errorCallback: Function,
})

Reply:
+1